XenAdminTests/UnitTests/AlertTests/AlertClassUnitTestData - Copy.cs

```diff
@@ -0,0 +1,38 @@
+﻿/* Copyright (c) Citrix Systems, Inc. 

```

Comment:
Generally we would want a file named `UpdateAlertClassUnitTestData.cs` rather than `AlertClassUnitTestData - Copy.cs`, but because it is such a small class I'd just put it in the same file as `AlertClassUnitTestData.cs`, it keeps the solution size smaller and makes navigation easier for the developer.